local_docs/fluid.list2buf.maxref.xml

```diff
@@ -35,6 +35,9 @@ Be aware that if a buffer of a predefined size is not supplied as the output it
     <outletlist>
     </outletlist>
 
+    <objarglist>
+    </objarglist>
+
```

Comment:
It'll be worth comparing mine and your XMLs for this file too: e.g. I have some docs for the `list` method and, more stuff in the `description` for the various bits

Reply:
Ah... it was commented out I think!

<img width="855" alt="image" src="https://user-images.githubusercontent.com/13281862/172487459-5f74be75-b803-4c5e-a104-00d04cce829d.png">

Reply:
Added this now in https://github.com/flucoma/flucoma-max/pull/250/commits/bdaed46a8100cd2eeae31a714bce0df470aa42a8